cfme/tests/services/test_myservice.py

```diff
@@ -208,7 +210,27 @@ def test_service_state():
             3.
             4. Service State should be Provisioned or Failed
     """
-    pass
+    service_catalogs = ServiceCatalogs(appliance, catalog_item.catalog, catalog_item.name)
+    service = MyService(appliance, catalog_item.name)
+    service_request = service_catalogs.order()
```

Comment:
You can use fixture 'order_service' that will order catalog_item and also delete service at the end.

https://github.com/ManageIQ/integration_tests/blob/master/cfme/fixtures/service_fixtures.py#L161

Reply:
@sshveta  we need to fail ordered service catalog item before successfully provisioned. so for that, we are removing provider while ordered catalog item is in progress hence we can not use fixture which provides successfully provision service catalog item.